t/watcher.t

```diff
@@ -235,3 +235,53 @@ SKIP: {
 
     is($watchresult, "CLIENT_ERROR watch commands not allowed\r\n", "attempted watch gave client error with no_watch option set");
 }
+
+
+# test delete/meta-delete with value sizes
+{
+    my $watcher = $server->new_sock;
+    print $watcher "watch deletions\n";
+    is(<$watcher>, "OK\r\n", "deletions watcher enabled");
+
+    print $client "set vfoo 0 0 4\r\nvbar\r\n";
+    is(<$client>, "STORED\r\n", "stored the key");
+
+    # wouldn't be logged to watcher
+    print $client "md non-existing-key\r\n";
+    is(<$client>, "NF\r\n", "non-existing key can't be deleted");
+
+    print $client "delete vfoo\r\n";
+    is(<$client>, "DELETED\r\n", "key was deleted");
+
+    sleep 1;
```

Comment:
has this sleep been necessary in your testing? the like command should block until a log line shows up

Reply:
I actually didn't try it without sleep. will see.

Reply:
without sleep works